<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myMainWindow</class>
 <widget class="QMainWindow" name="myMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actDoc_New"/>
   <addaction name="actDoc_Open"/>
   <addaction name="actClose_All"/>
   <addaction name="separator"/>
   <addaction name="actCut"/>
   <addaction name="actCopy"/>
   <addaction name="actPaste"/>
   <addaction name="actFont"/>
   <addaction name="separator"/>
   <addaction name="actMDI_Model"/>
   <addaction name="actCasde_Open"/>
   <addaction name="actArea_Open"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
  </widget>
  <action name="actDoc_New">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/new.jfif</normaloff>:/image/image/new.jfif</iconset>
   </property>
   <property name="text">
    <string>新建文档</string>
   </property>
  </action>
  <action name="actDoc_Open">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/open.jfif</normaloff>:/image/image/open.jfif</iconset>
   </property>
   <property name="text">
    <string>打开文档</string>
   </property>
  </action>
  <action name="actClose_All">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/close.jfif</normaloff>:/image/image/close.jfif</iconset>
   </property>
   <property name="text">
    <string>关闭全部</string>
   </property>
  </action>
  <action name="actCut">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/cut.jfif</normaloff>:/image/image/cut.jfif</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/copy.jfif</normaloff>:/image/image/copy.jfif</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/paste.jfif</normaloff>:/image/image/paste.jfif</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actFont">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/font.jfif</normaloff>:/image/image/font.jfif</iconset>
   </property>
   <property name="text">
    <string>字体设置</string>
   </property>
  </action>
  <action name="actMDI_Model">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/toolbarlab.jfif</normaloff>:/image/image/toolbarlab.jfif</iconset>
   </property>
   <property name="text">
    <string>MDI模式</string>
   </property>
  </action>
  <action name="actCasde_Open">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/Redo.jpg</normaloff>:/image/image/Redo.jpg</iconset>
   </property>
   <property name="text">
    <string>级联展开</string>
   </property>
  </action>
  <action name="actArea_Open">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/Undo.png</normaloff>:/image/image/Undo.png</iconset>
   </property>
   <property name="text">
    <string>平铺展开</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/redX.jpg</normaloff>:/image/image/redX.jpg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
